File: Python Code/Daily_Flash_Code_0/Week10/26Mar/57-DailyFlash_Solutions/26_Mar_Solution_Three/Java/prog3.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int num;
		Scanner sc = new Scanner(System.in);
		num = sc.nextInt();
		int [] arr = new int[num];
		for(int i = 0; i < num; i++) {
			arr[i] = sc.nextInt();
		}

		for(int j = 0; j < num / 2; j++) {
			
			int temp = arr[num - 1 - j];
			arr[num - 1 - j] = arr[j];
			arr[j] = temp;
		}
		for(int j = 0; j < num; j++) {
			System.out.print(arr[j]);
		}

	}
}

File: Python Code/Daily_Flash_Code_0/Week10/26Mar/57-DailyFlash_Solutions/26_Mar_Solution_Three/Python/prog3.py
import array
n = int(input())
arr = array.array("i", [])
j = 0
while(j < n):
    num = int(input())
    arr.append(num)
    j+=1
temp = arr[n - 1]
arr[n - 1] = arr[0]
arr[0] = temp

for i in range(len(arr)): 
    temp = arr[n - 1 - i]
    arr[n - 1 - i] = arr[i]
    arr[i] = temp
for i in arr:
    print(i, end = "")
